contracts/interfaces/ICoverProducts.sol

```diff
@@ -17,7 +17,9 @@ struct ProductInitializationParams {
 
 struct Product {
   uint16 productType;
-  address yieldTokenAddress;
+  uint16 minPrice;
+  // leftover memory gap from the previously used address field yieldTokenAddress
+  bytes18 __gap;
```

Comment:
can we use a `uint144` instead? it's easier when everything's in bits instead of bytes - no conversions needed to calculate

Reply:
Agree, it will be easier to make changes in tests in the future (in smart contracts doesn't matter as it should be never used). Fixed.